docs/learning/frontend/QuantityFormatting.md

```diff
@@ -336,6 +336,10 @@ parseResult.value // 7.5 (feet)
 #### Limitations
 Only plus(`+`) and minus(`-`) signs are supported for now.
 Other operators will end up returning a parsing error or an invalid input result.
+If a Format uses a spacer that conflicts with one of the operators above, additional restrictions will apply:
+
+- Mathematical operations only apply when it is in between whitespace. So `-2FT 6IN + 6IN` is equal to `-2FT-6IN + 6IN`, and `-2FT-6IN - 6IN` is not equal to `-2FT-6IN-6IN`.
+- For a value like `2FT 6IN-0.5`, the `-` sign will be treated as a spacer and not subtraction. However, the `0.5` value will use the default unit conversion provided to the parser, because it's not a part of the composite unit when that composite is made up of only 2 units - `FT` and `IN`.
```

Comment:
I feel that this is confusing. Can you provide examples of what the final result would be with different possible parser settings?

Reply:
I've added examples to each of the bullet points, encased in a markdown expandable block. I haven't seen the expandable blocks used in iTwinjs.org before, so @ben-polinsky lmk if it doesn't look like it'll work

Reply:
See example [here](https://www.itwinjs.org/presentation/advanced/localization/#on-the-backend)

Reply:
i checked the code for that .md file, the syntax is the same as my use here, i think we're good to go